ticdc: update storage sink doc (#14510)
pingcap/docs: ticdc/ticdc-sink-to-cloud-storage.md

> **Note:**
>

> > The table version changes only after a DDL operation is performed for the upstream table: the table version is the TSO when the upstream TiDB completes the execution of the DDL. However, the change of the table version does not mean the change of the table schema. For example, adding a comment to a column does not cause the schema file content to changed.

Author: ```suggestion
> The table version changes only after a DDL operation is performed on the upstream table, and the new table version is the TSO when the upstream TiDB completes the execution of the DDL. However, the change of the table version does not mean the change of the table schema. For example, adding a comment to a column does not cause the schema file content to change.
```

### DDL events

### DDL events at the table level

When a DDL event of an upstream table causes a table version change, TiCDC automatically does the following:

- Changes to a new path to write data change records. For example, when the version of `test.table1` changes to `441349361156227074`, TiCDC changes to the `s3://bucket/bbb/ccc/test/table1/441349361156227074/2022-01-02/` the path to write data change records.

> - Generate a schema file in the following path to store the table schema information:

Author: ```suggestion
- Generates a schema file in the following path to store the table schema information:
```

    ```shell
    {scheme}://{prefix}/{schema}/{table}/meta/schema_{table-version}_{hash}.json
    ```

> Taking `schema_441349361156227074_3131721815.json` as an example, the content in the table schema information file is as follows:

Author: ```suggestion
Taking the `schema_441349361156227074_3131721815.json` schema file as an example, the table schema information in this file is as follows:
```

    - `ColumnIsPk`: The column is part of the primary key when the value of this option is `true`.
- `TableColumnsTotal`: The size of the `TableColumns` array.

### DDL events at the database level

When a database-level DDL event is performed in the upstream database, TiCDC automatically generates a schema file in the following path to store the database schema information:

```shell
{scheme}://{prefix}/{schema}/meta/schema_{table-version}_{hash}.json
```

> Taking `schema_441349361156227000_3131721815.json` as an example, the content of the database schema information file is as follows:

Author: ```suggestion
Taking the `schema_441349361156227000_3131721815.json` schema file as an example, the database schema information in this file is as follows:
```

> Taking the `schema_441349361156227000_3131721815.json` schema file as an example, the database schema information in this file is as follows:

Reviewer: 中文版也需要改下，schema 改为 schema1
```suggestion
Taking the `schema1_441349361156227000_3131721815.json` schema file as an example, the database schema information in this file is as follows:
```

Reviewer 2: 为什么要改为 schema1 呢？

Reviewer 2: ![img_v2_94526e93-bb3d-4ec2-be50-6d073c9d622g](https://github.com/pingcap/docs/assets/61726649/1d922215-e004-4e79-a969-88326d1ab606)
文件名与文件内容无关的

```json
{
  "Table": "",
  "Schema": "schema1",
  "Version": 1,
  "TableVersion": 441349361156227000,
  "Query": "CREATE DATABASE `schema1`",
  "Type": 1,
  "TableColumns": null,
  "TableColumnsTotal": 0
}
```

### Data type

> This section describes the data types used in the `schema_{table-version}_{hash}.json` file (shorten as schema file). The data types are defined as `T(M[, D])`. For details, see [Data Types](/data-type-overview.md).

Author: ```suggestion
This section describes the data types used in the `schema_{table-version}_{hash}.json` file (shorten as schema file in the following sections). The data types are defined as `T(M[, D])`. For details, see [Data Types](/data-type-overview.md).
```

> This section describes the data types used in the `schema_{table-version}_{hash}.json` file (shorten as schema file in the following sections). The data types are defined as `T(M[, D])`. For details, see [Data Types](/data-type-overview.md).

Reviewer: ```suggestion
This section describes the data types used in the `schema_{table-version}_{hash}.json` file (hereafter referred to as "schema file" in the following sections). The data types are defined as `T(M[, D])`. For details, see [Data Types](/data-type-overview.md).
```
